### DISCUSSION TITLE
"About" section

### DISCUSSION THREAD

**Discussion by Tharindu37** [Tharindu37/ProSolutions#68] [Ideas]
Will we add our team members as developers in the "ProSolution" website to the "about" section?

**Comment by Tharindu37** [comment]
@BawanthaPrasad 
@AK4266 
@Navodi99 
@emergenitro 
@aayushdhakal
@hiruhemapala
@khushpreetsinghb
@Yasanga-Nishaditha
@shiva0002

**Reply by AK4266** [reply]
Yeah

**Reply by Tharindu37** [reply]
@AK4266 So, for this want to get the profile picture and name. Can you provide the way for it?

**Reply by AK4266** [reply]
Yes, I'll think about it and let you know

**Reply by Tharindu37** [reply]
@AK4266 Oky

**Reply by BawanthaPrasad** [reply]
Yep, that's better, So about section should be Redesigned?

**Reply by Tharindu37** [reply]
@BawanthaPrasad no, We will Add the profile pictures after the currnt "about" section.

**Reply by aayushdhakal** [reply]
we can simply use the git api to call the profile name as well as the picture . we can implement that way @Tharindu37

**Reply by Tharindu37** [reply]
@aayushdhakal Yep, It's a good idea. Let's do that.

**Comment by Tharindu37** [comment]
I added the Contributor section as follows. What do you think about it?
![Screenshot 2023-03-16 105742](https://user-images.githubusercontent.com/88996660/225523871-93a67628-bb28-4412-a0b8-f9199bad9a2d.jpg)
@BawanthaPrasad
@AK4266
@Navodi99
@emergenitro
@aayushdhakal
@hiruhemapala
@khushpreetsinghb
@Yasanga-Nishaditha
@shiva0002

**Reply by AK4266** [reply]
It's good 🔥

**Reply by Tharindu37** [reply]
@AK4266 Thank you. You can add the new issues if you can see any issues.

**Reply by aayushdhakal** [reply]
It's good.
are we going to add this on about section or completly new page ? @Tharindu37

**Reply by BawanthaPrasad** [reply]
It's good.I prefer them in centered align and if we could add a boarder around profile picture.

**Reply by Tharindu37** [reply]
@aayushdhakal Let's create a new page for the "solutions"  section

**Reply by Tharindu37** [reply]
@BawanthaPrasad It's a good idea. can you create a new issue for Add add a border around the profile Picture?

**Reply by Tharindu37** [reply]
@BawanthaPrasad Can you design the "Solutions" page using any tool?

**Reply by aayushdhakal** [reply]
@Tharindu37  can you please explain about the solution page.

**Reply by Tharindu37** [reply]
@aayushdhakal I'll think and tell

**Reply by aayushdhakal** [reply]
sure

**Reply by BawanthaPrasad** [reply]
> @BawanthaPrasad It's a good idea. can you create a new issue for Add add a border around the profile Picture?

@Tharindu37 ok, I'll
